### ISSUE TITLE
'[WinError 126] The specified module could not be found' when trying to use uvicorn server:app reload

### ISSUE THREAD

**Issue by CrazyMew37** [darabos/ai-alchemy#3]
Hello, I'm currently having an issue with running the game. I managed to run the first command as well as comfyui successfully but when I try to run the server itself, all I get is this error.
![image](https://github.com/user-attachments/assets/70c02160-b414-43b8-828b-b52b91fadf87)
How exactly can I solve this?

**Comment by darabos** [comment]
Hi! The error message looks like you got `ctranslate2` installed, but the Python module can't find the corresponding binary module (the DLL file it's trying to load with `ctypes`). You could check `c:\pandas3d-1.10.14-x86\python\lib\site-packages\ctranslate2` to see if the DLL is there. But this whole path looks like you're using the Python that's come with Panda3D. It may be better to use a separate Python installation.

I recommend `uv` (https://docs.astral.sh/uv/). It lets you set up and manage Python installations easily, and it's super fast. But stock Python from https://python.org/ should be fine too.

**Comment by CrazyMew37** [comment]
I checked the folder, and this is what it looks like:
![image](https://github.com/user-attachments/assets/22aebee6-9a46-464e-ae2d-113731289226)
I've installed UV and will try to install a new version of python. Which version do you recommend?

**Comment by darabos** [comment]
> I checked the folder, and this is what it looks like:

Right, the `ctranslate2.dll` is there. It's just not found because the paths are not set up right.

> I've installed UV and will try to install a new version of python. Which version do you recommend?

I'm not sure what I used with this project. I'd try 3.11.

**Comment by CrazyMew37** [comment]
Good news: the UV virtual environment is the way to go! Only issue is that after a few headaches, I've hit another brick wall (And it's with ctranslate2 again)
![image](https://github.com/user-attachments/assets/ac5857a9-a1f6-44d0-a6e3-b60c4a3fe7b1)
Judging from the traceback, apparently, some dll is missing? I checked the folder where it's in, and the dll is indeed missing.
![image](https://github.com/user-attachments/assets/adc636f5-c028-4df1-846e-63e491ce0ee8)
(Just to not confuse you, I'm using Windows 11 23H2 that looks like Windows XP, that's why it looks off.)
